javascript/javascript1/week2/event_application.js

```diff
@@ -0,0 +1,14 @@
+
+    
+    const daysOfWeek=['MOnday','Tuesday','Wednsday','Thursday','Friday','Saturday','Sunday'];
+    function getEventWeekday(futureEventdayNumber){
+    const today= new Date();
+    const dayOfEvent=((today.getDay() + futureEventdayNumber)%7)-1; //since array index starts from 0 
```

Comment:
Indenting is wrong here. These lines should vistually be "inside" the function (same indenting as `return`).